Zend/Optimizer/zend_call_graph.c

```diff
@@ -107,6 +107,8 @@ ZEND_API int zend_analyze_calls(zend_arena **arena, zend_script *script, uint32_
 			case ZEND_DO_UCALL:
 			case ZEND_DO_FCALL_BY_NAME:
 				func_info->flags |= ZEND_FUNC_HAS_CALLS;
+			    /* intentional fall through */
```

Comment:
I'm not sure about this change, but this seemed reasonable ?

Reply:
I think you can just treat partial the same as anything else here -- especially as the ZEND_FUNC_HAS_CALLS flag is completely unused, so it's hard to say what the correct behavior for partials is supposed to be...